File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        for(int i=0;i<nums.size();i++){
            int findVal = target-nums[i];
            for(int j=i+1;j<nums.size();j++){
                if(nums[j] == findVal){
                    return vector<int>{i,j};
                }
            }
        }
        
    return vector<int>{1,2};
    }
};

File: 0001-two-sum/0001-two-sum.php
class Solution {

    /**
     * @param Integer[] $nums
     * @param Integer $target
     * @return Integer[]
     */
    function twoSum($nums, $target) {
        $map = [];
        foreach($nums as $index => $num){
            $rem = $target-$num;
            if(isset($map[$rem])) return[$index,$map[$rem]];
            $map[$num] = $index;
        }
    }
}